source/Octopus.Client.Tests/Octopus.Client.Tests.csproj

```diff
@@ -12,18 +12,18 @@
     <LangVersion>8</LangVersion>
   </PropertyGroup>
   <PropertyGroup Condition="!$([MSBuild]::IsOSUnixLike())">
-    <TargetFrameworks>net462;net48;net6.0</TargetFrameworks>
+    <TargetFrameworks>net462;net48;net6.0;net8.0</TargetFrameworks>
   </PropertyGroup>
   <PropertyGroup Condition="$([MSBuild]::IsOSUnixLike())">
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
   </PropertyGroup>
```

Comment:
@borland Do we still need to target .NET 6?

Reply:
Yeah, if the library targets net6 then the tests should too